httpx/client.py

```diff
@@ -222,6 +223,25 @@ def params(self, params: QueryParamTypes) -> None:
         timeout: typing.Union[TimeoutTypes, UnsetType] = UNSET,
         trust_env: bool = None,
     ) -> Response:
+        if cert is not None:
+            warnings.warn(
+                "Passing a 'cert' argument when making a request on a client "
+                "is due to be deprecated. Instantiate a new client instead, "
+                "passing any 'cert' arguments to the client itself."
+            )
+        if verify is not None:
+            warnings.warn(
+                "Passing a 'verify' argument when making a request on a client "
+                "is due to be deprecated. Instantiate a new client instead, "
+                "passing any 'verify' arguments to the client itself."
+            )
+        if trust_env is not None:
+            warnings.warn(
+                "Passing a 'trust_env' argument when making a request on a client "
+                "is due to be deprecated. Instantiate a new client instead, "
+                "passing any 'trust_env' argument to the client itself."
+            )
```

Comment:
Same category suggestion.
```suggestion
            , PendingDeprecationWarning)
```